gvfs-helper.c

```diff
@@ -3800,6 +3807,8 @@ static enum gh__error_code do_sub_cmd__prefetch(int argc, const char **argv)
 		if (my_parse_since(since_str, &seconds_since_epoch))
 			die("could not parse 'since' field");
 	}
+	if (gh__cmd_opts.max_retries < 0)
+		gh__cmd_opts.max_retries = 0;
```

Comment:
Are we going to start passing this recursively so this actually decreases as we go?

Reply:
I see from looking at the file: `gh__cmd_opts.max_retries` is used properly in lots of places, just wasn't used in 'prefetch' yet. Excellent.